src/storage/src/statistics.rs

```diff
@@ -132,6 +134,77 @@ impl SourceStatisticsMetrics {
     }
 }
 
+#[derive(Clone, Debug)]
+pub(crate) struct SinkStatisticsMetricsDefinitions {
+    pub(crate) messages_staged: IntCounterVec,
+    pub(crate) bytes_staged: IntCounterVec,
+    pub(crate) messages_committed: IntCounterVec,
+    pub(crate) bytes_committed: IntCounterVec,
+}
+
+impl SinkStatisticsMetricsDefinitions {
+    pub(crate) fn register_with(registry: &MetricsRegistry) -> Self {
+        Self {
+            messages_staged: registry.register(metric!(
```

Comment:
Taking a step back: these metrics names can be somewhat confusing now. There's `mz_updates_staged` (for sources), and `mz_messages_staged` (for sinks). Should we maybe prefix all of these with `mz_source_`/`mz_sink`?

Reply:
yeah, I can clean these up!